src/sort_deck_of_cards/sort_deck_of_cards.py

```diff
@@ -0,0 +1,18 @@
+"""Kata Sort deck of cards."""
+
+
+def sort_cards(cards):
```

Comment:
This works, but is highly inefficient.  Try less if/then checking and less looping.  Look into sorted with a key parameter.